.github/workflows/main.yml

```diff
@@ -91,6 +109,13 @@ jobs:
           key: ${{ runner.os }}-${{ hashFiles('*/external/binaries-list', '*/*/external/binaries-list') }}
           restore-keys: ${{ runner.os }}-
 
+      - name: Setup Gradle Daemon to run on JDK 11
+        if: ${{ matrix.java > 18 }}
+        run: |
+          mkdir -p ~/.gradle
+          #uses a preinstalled JDK 11 from the runner
+          echo "org.gradle.java.home=$JAVA_HOME_11_X64" >> ~/.gradle/gradle.properties
```

Comment:
this allows us to build on JDK 19+ too.

Its unclear why building locally on JDK 19 works perfectly fine without having to configure the daemon to run somewhere else.